HotKey/HotKey_MainFolder/HotKeyControl.resx

```diff
@@ -0,0 +1,120 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I like the choice of WPF definitely looks a lot cleaner than winforms. Nice idea.